recipes/pyferret/site_specific.patch

```diff
@@ -0,0 +1,183 @@
+diff -Naur external_functions/ef_utility/platform_specific.mk.x86_64-linux external_functions/ef_utility/platform_specific.mk.x86_64-linux
+--- external_functions/ef_utility/platform_specific.mk.x86_64-linux	2016-06-17 21:09:23.000000000 -0300
++++ external_functions/ef_utility/platform_specific.mk.x86_64-linux	2016-06-22 18:03:51.357859794 -0300
+@@ -23,7 +23,7 @@
+ LD		= gfortran
+ LD_DYN_FLAGS	= -fPIC -m64 -shared -Wl,--no-undefined
+ SYSLIBS		= ../ef_utility/copy_ferret_ef_mem_subsc.o \
+-		  -L$(INSTALL_FER_DIR)/lib64/$(PYTHON_EXE)/site-packages/pyferret \
++		  -L$(SP_DIR)/pyferret \
+ 		  -L$(INSTALL_FER_DIR)/lib/$(PYTHON_EXE)/site-packages/pyferret \
+ 		  -lpyferret
+ 
+diff -Naur external_functions/ef_utility/site_specific.mk external_functions/ef_utility/site_specific.mk
+--- external_functions/ef_utility/site_specific.mk	2016-06-17 21:09:23.000000000 -0300
++++ external_functions/ef_utility/site_specific.mk	2016-06-22 15:41:24.742434146 -0300
+@@ -27,7 +27,7 @@
+ INSTALL_FER_DIR = $(FER_DIR)
+ 
+ ## Python version used by PyFerret, either python2.6 or python2.7
+-PYTHON_EXE = python2.6
++PYTHON_EXE = python$(PY_VER)
+ # PYTHON_EXE = python2.7
+ 
+ ## FER_LOCAL_EXTFCNS is the directory in which to install
+diff -Naur Makefile Makefile
+--- Makefile	2016-06-17 21:09:23.000000000 -0300
++++ Makefile	2016-06-22 18:04:31.057861771 -0300
+@@ -54,9 +54,10 @@
+ 	  export PIXMAN_LIBDIR=$(PIXMAN_LIBDIR) ; \
+ 	  export HDF5_LIBDIR=$(HDF5_LIBDIR) ; \
+ 	  export NETCDF4_LIBDIR=$(NETCDF4_LIBDIR) ; \
+-	  $(PYTHON_EXE) setup.py --quiet build )
++	  $(PYTHON_EXE) setup.py --quiet build ; \
++	  $(PYTHON_EXE) setup.py install )
```

Comment:
I had to perform a install to "force" it to find the lib in a known place. Otherwise it was the temporary `buil/lib.$arch$os` directory.